docker-compose.yml

```diff
@@ -10,13 +10,19 @@ services:
     tty: true
     volumes:
       - ./.env:/home/node/app/.env
+    environment:
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
```

Comment:
There are 2 other environments that you might want to change as well for the `relay` to actually use the relay instance.

https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/docs/configuration.md
`REDIS_ENABLED` and `REDIS_URL`, don't think that the `REDIS_HOST` and `REDIS_PORT` are defined as configurations on the `relay`, shared the link with the configuration doc for your reference. Again, seems to be working due to the default values.

Reply:
done, please check